docs/Architecture/CBS.md

```diff
@@ -0,0 +1,69 @@
+## Current Processes
+
+### CBS: pulls data from email to s3
+- **Environment Variables**: MAILPASS, MAILUSER
+- **Relevant input**: input emails from CBS containing zip files
+- **Relevant output**: unzipped CBS files saved in S3
+- **Relevant Storage directory**: S3 -> Bucket: anyway-cbs
+- **command**: `python3 main.py scripts importemail`
```

Comment:
Maybe add where the code resides.

Reply:
@ziv17 what do you mean? In the scheduling section below it says it's in Jenkins

Reply:
I mean the python code, file or directory or group of files (e.g. infographics*)